src/main/java/io/confluent/connect/elasticsearch/ElasticsearchSinkConnectorConfig.java

```diff
@@ -19,18 +19,23 @@
 import org.apache.kafka.common.config.ConfigDef.Importance;
 import org.apache.kafka.common.config.ConfigDef.Type;
 import org.apache.kafka.common.config.ConfigDef.Width;
+import org.apache.kafka.common.config.internals.BrokerSecurityConfigs;
 
+import java.util.List;
 import java.util.Map;
 
 import static io.confluent.connect.elasticsearch.DataConverter.BehaviorOnNullValues;
 import static io.confluent.connect.elasticsearch.bulk.BulkProcessor.BehaviorOnMalformedDoc;
+import static org.apache.kafka.common.config.SslConfigs.addClientSslSupport;
 
 public class ElasticsearchSinkConnectorConfig extends AbstractConfig {
 
   public static final String CONNECTION_URL_CONFIG = "connection.url";
   private static final String CONNECTION_URL_DOC =
       "List of Elasticsearch HTTP connection URLs e.g. ``http://eshost1:9200,"
-      + "http://eshost2:9200``.";
+      + "http://eshost2:9200``, or ``https://eshost3:9200``. If any of the URLs specifies "
+      + "``https:`` protocol, https will be used for all connections. A URL with no protocol will "
+      + "be treated as ``http``.";
```

Comment:
Nit: the original doc string wasn't terribly useful, but this could still be cleaned up a bit:
```
The comma-separated list of one or more Elasticsearch URLs, such as
``http://eshost1:9200,http://eshost2:9200`` or ``https://eshost3:9200``. 
HTTPS will be used for all connections if any of the URLs start with ``https:``. 
A URL without a protocol will be treated as ``http``.
```